libs/designsystem/src/lib/components/item-sliding/item-sliding.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, Input } from '@angular/core';
+
+import { ItemSlidingSide, ItemSwipeAction } from './item-sliding.types';
+
+@Component({
+  selector: 'kirby-item-sliding',
+  templateUrl: './item-sliding.component.html',
+  styleUrls: ['./item-sliding.component.scss'],
+})
+export class ItemSlidingComponent {
+  @Input() swipeActions: ItemSwipeAction[];
+
+  _side: 'start' | 'end' = 'start';
+  @Input() set side(value: ItemSlidingSide) {
+    this._side = value === 'left' ? 'start' : 'end';
+  }
+
+  get _hasSwipeActions(): boolean {
+    return Array.isArray(this.swipeActions) && this.swipeActions.length !== 0;
```

Comment:
You're completely right. It's actually a bit of leftover code from when i was goofing around with it.

Reply:
Ah remembered why i did it. 

The alternative should be to do something like this: 
```typescript
this.swipeActions?.length !== 0;
```

This is however true in the case that `this.swipeActions` is undefined. As `undefined !== 0` is a true statement. 

But what can be done instead is: 

```typescript
   return this.swipeActions?.length > 0; 
```

as `undefined > 0` is false. 

In my head at least, i think that for future developers that might look at this, it might not be clear why larger than is used instead of not equals. Therefore to express intention Array.isArray() is used. 

Buuuut there's no reason the computer should do extra work while it is completely negligble to express that intention when i could just throw in a comment instead. 

So that is what i will be doing 😄